packages/dx-react-scheduler-material-ui/src/plugins/day-view.jsx

```diff
@@ -4,29 +4,29 @@ import { VerticalViewLayout as Layout } from '../templates/layouts/vertical-view
 
 import { Row } from '../templates/views/common/row';
 import { DayScaleEmptyCell } from '../templates/views/vertical/day-scale-empty-cell';
-import { Container } from '../templates/appointment/container';
+import { Container as AppointmentLayer } from '../templates/appointment/container';
```

Comment:
Should we also rename the file name of a component?

Reply:
I'm not sure. It will look a bit strange - `/templates/appointment/layer`. What layer?) I'd prefer `Container` in a context of the `appointment` folder. However, `Layer` in a context of a view.